src/azure-cli-core/azure/cli/core/_help.py

```diff
@@ -40,6 +40,7 @@
 
 Welcome to the cool new Azure CLI!
 
+Use `az --version` to display the current version of the cli and modules.
```

Comment:
Can you remove ` of the cli and modules`?

Reply:
@derekbekoe I've updated per your requet